ansible_collections/arista/avd/README.md

```diff
@@ -47,51 +32,57 @@ The arista.avd collection provides abstracted data models and framework to build
 
 ## Roles Overview
 
-This repository provides roles for Ansible's collection __arista.avd__ with the following content:
+This repository provides roles for Aristas's collection, **arista.avd**, with the following content:
```

Comment:
@tgodaA I feel like this could be written as "This repository provides content for Arista's **arista.avd** collection. The following roles are included.